# HSS Koordinat Dizisi
Header header
HssKoordinat[] hss_koordinat_bilgileri
